### DISCUSSION TITLE
Pull Request number

### DISCUSSION THREAD

**Discussion by avigoldman** [vercel/vercel#5479] [Help] [answered]
I'm trying to compute a render.com preview url which requires knowing the pull request number. Is there a way to access the pull request number in the build step?

**Comment by mcsdevv** [comment]
Hey @avigoldman, I don't believe we make the specific PR number available, but you may be able to find something that works here: https://vercel.com/docs/platform/environment-variables#system-environment-variables

**Reply by avigoldman** [reply]
Gotcha - I ended up creating a little CLI to find the related PR number 😊

https://github.com/useparcel/vercel-is-pull-request

**Reply by lardissone** [reply]
+1 to include the PR number by default!

Thanks @avigoldman for the temp solution!

**Reply by JClackett** [reply]
Would love this, Railway uses the PR number in their preview urls so if I want to use Next.js preview deployments on a PR, Ill need to dynamically access the correct API URL

**Reply by mikecousins** [reply]
Heroku uses the PR number as well. Native support for this would be much appreciated. It's the one really annoying thing missing since moving from Netlify to Vercel.

**Reply by amyegan** [reply, marked as answer]
Pull request number is now available as `VERCEL_GIT_PULL_REQUEST_ID` in [System Environment Variables](https://vercel.com/docs/concepts/projects/environment-variables/system-environment-variables#system-environment-variables)

**Reply by avigoldman** [reply]
Amazing, I'm going to mark the CLI as deprecated!
